diff --git a/vendor/silversolutions/silver.e-shop/src/Silversolutions/Bundle/EshopBundle/Resources/views/pagelayout.html.twig b/vendor/silversolutions/silver.e-shop/src/Silversolutions/Bundle/EshopBundle/Resources/views/pagelayout.html.twig
index 4f9b0f9b..9f86c992 100644
--- a/vendor/silversolutions/silver.e-shop/src/Silversolutions/Bundle/EshopBundle/Resources/views/pagelayout.html.twig
+++ b/vendor/silversolutions/silver.e-shop/src/Silversolutions/Bundle/EshopBundle/Resources/views/pagelayout.html.twig
@@ -82,7 +82,7 @@
 
         {% block language_switcher_mobile %}
           <li class="u-padding-1x clearfix">
-
+            {#
             <a
               class="c-nav-meta__flag{% if current_siteaccess != 'en' %} is-current{% endif %} js-link-language-switcher"
               href="{{ url('silversolutions_language_switcher', {'siteaccess' : 'de', 'source' : source, 'id' : id, 'query_string' : app.request.queryString }) }}"
@@ -95,9 +95,10 @@
                 <path fill="#d00" d="M0 170.67h512.005v170.668H0z"/>
               </svg>
             </a>
+            #}
             <a
-              class="c-nav-meta__flag{% if current_siteaccess == 'en' %} is-current{% endif %} js-link-language-switcher"
-              href="{{ url('silversolutions_language_switcher', {'siteaccess' : 'en', 'source' : source, 'id' : id, 'query_string' : app.request.queryString }) }}"
+              class="c-nav-meta__flag{% if current_siteaccess == 'site' %} is-current{% endif %} js-link-language-switcher"
+              href="{{ url('silversolutions_language_switcher', {'siteaccess' : 'site', 'source' : source, 'id' : id, 'query_string' : app.request.queryString }) }}"
               title="USA">
               <svg class="c-nav-meta__flag-icon" xmlns="http://www.w3.org/2000/svg" height="22" width="22"
                    viewBox="0 0 512 512">
@@ -181,9 +182,10 @@
 
                   {% block language_switcher %}
                     <li>
+                      {#
                       <a
                         class="c-nav-meta__flag{% if current_siteaccess != 'en' %} is-current{% endif %} js-link-language-switcher"
-                        href="{{ url('silversolutions_language_switcher', {'siteaccess' : 'de', 'source' : source, 'id' : id, 'query_string' : app.request.queryString }) }}"
+                        href="{{ url('silversolutions_language_switcher', {'siteaccess' : 'site', 'source' : source, 'id' : id, 'query_string' : app.request.queryString }) }}"
                         title="Deutschlandland">
                         <svg class="c-nav-meta__flag-icon" xmlns="http://www.w3.org/2000/svg" height="22" width="22"
                              viewBox="0 0 512 512">
@@ -193,9 +195,10 @@
                           <path fill="#d00" d="M0 170.67h512.005v170.668H0z"/>
                         </svg>
                       </a>
+                      #}
                       <a
-                        class="c-nav-meta__flag{% if current_siteaccess == 'en' %} is-current{% endif %} js-link-language-switcher"
-                        href="{{ url('silversolutions_language_switcher', {'siteaccess' : 'en', 'source' : source, 'id' : id, 'query_string' : app.request.queryString }) }}"
+                        class="c-nav-meta__flag{% if current_siteaccess == 'site' %} is-current{% endif %} js-link-language-switcher"
+                        href="{{ url('silversolutions_language_switcher', {'siteaccess' : 'site', 'source' : source, 'id' : id, 'query_string' : app.request.queryString }) }}"
                         title="USA">
                         <svg class="c-nav-meta__flag-icon" xmlns="http://www.w3.org/2000/svg" height="22" width="22"
                              viewBox="0 0 512 512">
@@ -465,12 +468,13 @@
   storm_translations.set(translations);
 </script>
 
-
+{#
 {% if app.environment == 'dev' %}
   <script type='text/javascript' id="__bs_script__">//<![CDATA[
     document.write("<script async src='http://HOST:3000/browser-sync/browser-sync-client.js?v=2.18.2'><\/script>".replace("HOST", location.hostname));
     //]]></script>
 {% endif %}
+#}
 
 {% block tracker_bottom_base %}
   {{ tracker.bottom(tracking_codes_base) }}
