diff --git a/vendor/silversolutions/silver.e-shop/src/Silversolutions/Bundle/EshopBundle/Controller/CatalogController.php b/vendor/silversolutions/silver.e-shop/src/Silversolutions/Bundle/EshopBundle/Controller/CatalogController.php
index f475b750..a252740d 100644
--- a/vendor/silversolutions/silver.e-shop/src/Silversolutions/Bundle/EshopBundle/Controller/CatalogController.php
+++ b/vendor/silversolutions/silver.e-shop/src/Silversolutions/Bundle/EshopBundle/Controller/CatalogController.php
@@ -74,9 +74,10 @@ class CatalogController extends BaseController
         /** @var $urlService \Silversolutions\Bundle\EshopBundle\Services\Url\CatalogUrl */
         $urlService = $this->get('silver_catalog.url_service');
 
-        $url = rawurldecode($urlService->getSeoUrl($request));
+        $url = rawurldecode($request->getPathInfo());
 
-        if (strpos($request->attributes->get('semanticPathinfo'), '/content/versionview') === 0) {
+        if (strpos($request->attributes->get('semanticPathinfo'), '/content/versionview') === 0 ||
+            strpos($request->attributes->get('semanticPathinfo'), '/content/preview') === 0) {
             $location = $request->attributes->get('location');
             $locationId = $location->id;
             /** @var $catalogElement \Silversolutions\Bundle\EshopBundle\Catalog\CatalogElement */
@@ -88,6 +89,8 @@ class CatalogController extends BaseController
                     'content' => $request->attributes->get('content')),
                     $this->getLanguages());
         } else {
+            $url = rawurldecode($urlService->getSeoUrl($request));
+
             /** @var $catalogElement \Silversolutions\Bundle\EshopBundle\Catalog\CatalogElement */
             $catalogElement = $catalogService->getDataProvider()
                 ->lookupByUrl(rawurldecode($url), $this->getLanguages());
