diff --git a/vendor/silversolutions/silver.e-shop/src/Silversolutions/Bundle/EshopBundle/Twig/SilvercommonExtension.php b/vendor/silversolutions/silver.e-shop/src/Silversolutions/Bundle/EshopBundle/Twig/SilvercommonExtension.php
index 1a0d1493..486c70a1 100644
--- a/vendor/silversolutions/silver.e-shop/src/Silversolutions/Bundle/EshopBundle/Twig/SilvercommonExtension.php
+++ b/vendor/silversolutions/silver.e-shop/src/Silversolutions/Bundle/EshopBundle/Twig/SilvercommonExtension.php
@@ -826,9 +826,9 @@ class SilvercommonExtension extends Twig_Extension implements Twig_Extension_Glo
         array $params = array())
     {
         if ($field->text instanceof \DOMDocument) {
-            /** @var RichTextExtension $xmlTextExtension */
-            $xmlTextExtension = $this->container->get('ezpublish.twig.extension.rich_text');
-            return $xmlTextExtension->richTextToHtml5($field->text);
+            /** @var \eZ\Publish\Core\FieldType\RichText\Converter $richTextConverter */
+            $richTextConverter = $this->container->get('ezpublish.fieldType.ezrichtext.converter.output.xhtml5');
+            return $richTextConverter->convert($field->text)->saveHTML();
         }
 
         $view = sprintf('SilversolutionsEshopBundle:Fieldtypes:%s.html.twig', $this->extractFieldClassName($field));
